### PR TITLE
translate ja/chronos

### PR DESCRIPTION
**Pull request by gr-ishiguro**
I translating ja/chronos.

### CODE REVIEW

**Comment by markstory** [comment]
Looks like there is [a build error](https://travis-ci.org/cakephp/docs/builds/116411885#L276).

File: ja/chronos.rst [hunk 1]
```diff
@@ -1,13 +1,451 @@
...
+
+概要
+----
+
+..
+  Chronos provides a number of extensions to the DateTime objects provided by PHP.
+  Chronos provides 5 classes that cover mutable and immutable date/time variants
+  and extensions to ``DateInterval``.
+
+ChronosはPHPが提供するDateTimeオブジェクトの拡張の数値を提供します。
```
Comment (chinpei215) [inline ja/chronos.rst]: Probably, `a number of` means several, some.

Comment (gr-ishiguro) [inline ja/chronos.rst]: thank you

File: ja/chronos.rst [hunk 2]
```diff
@@ -1,13 +1,451 @@
...
+  data can be modified accidentally or without the developer knowing.
+  Immutable objects prevent accidental changes to
+  data, and make code free of order-based dependency issues. Immutability
+  does mean that you will need to remember to replace variables when using
+  modifiers::
+..
+      // This code doesn't work with immutable objects
+      // This works like you'd expect
+
+もし、PHPの ``DateTime`` オブジェクトを使用したい場合、 *ミュータブル* オブジェクトで簡単にできます。
```
Comment (chinpei215) [inline ja/chronos.rst]: have + past participle = Experience (〜したことがある)

Probably, the original sentence says that if you have experiences of using `DateTime`, it is easy to use *mutable* objects. 
(As `Cake\Chronos\MutableDate` is a subclass of `DateTime`, you can use it in the same way.)

File: ja/chronos.rst [hunk 3]
```diff
@@ -1,13 +1,451 @@
...
+* ``Cake\Chronos\MutableTime`` はミュータブルな *日付と時刻* オブジェクト。
+* ``Cake\Chronos\MutableDate`` はミュータブルな *日付* オブジェクト。
+* ``Cake\Chronos\ChronosInterval`` は ``DateInterval`` の拡張機能。
+
+..
+  Lastly, if you want to typehint against Chronos-provided date/time objects you
+  should use ``Cake\Chronos\ChronosInterface``. All of the date and time objects
+  implement this interface.
+
+最後に、もしあなたがChronosが提供する 日付/時刻 のオブジェクトに反した型宣言を行ないたい場合、
```
Comment (chinpei215) [inline ja/chronos.rst]: Probably, `against` should be translated as `対して`. I think the original sentence explains how to typehint. For example, `Cake\I18n\RelativeTimeFormatter::diffForHumans()` is declared as the following:
```php
public function diffForHumans(ChronosInterface $date, ChronosInterface $other = null, $absolute = false)
```

**Comment by okinaka** [comment]
@gr-ishiguro Thanks!

**Comment by chinpei215** [comment]
@gr-ishiguro Thank you for translating!

**Pull request merged**